app/src/main/java/org/kiwix/kiwixmobile/utils/KiwixDialog.kt

```diff
@@ -30,23 +30,30 @@ sealed class KiwixDialog(
   }
 
   object LocationPermissionRationale : KiwixDialog(
-      null, R.string.permission_rationale_location, android.R.string.yes, android.R.string.cancel
+    null, R.string.permission_rationale_location, android.R.string.yes, android.R.string.cancel
   )
 
   object StoragePermissionRationale : KiwixDialog(
-      null, R.string.request_storage, android.R.string.yes, android.R.string.cancel
+    null, R.string.request_storage, android.R.string.yes, android.R.string.cancel
   )
 
   object EnableWifiP2pServices : KiwixDialog(
```

Comment:
If a KiwixDialog ends in `yes no` you can instead switch the parent to be `YesNoDialog`, examples are further down the class